### DISCUSSION TITLE
I use the pro version but my account is still locked

### DISCUSSION THREAD

**Discussion by thanhdong66** [vercel/community#3257] [Help] [answered]
### Summary

I can't log in to my account.
how much pro account can i create 'Import Git Repository'

### Example

![Screenshot 2023-07-16 002545](https://github.com/vercel/community/assets/139622250/faa5af8f-3eef-41a3-ad0a-8e8d409fe466

**Comment by amyegan** [comment, marked as answer]
Hey @thanhdong66. Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
